bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,329 @@
+            title="Spook Name Rate",
+            description=f"Help on the `{self.bot.command_prefix}spooknamerate` command",
+            color=Colours.soft_orange,
+        )
+
+        emoji_message = ""
+        for emoji in self.emojis_val:
+            emoji_message += f"- {emoji} {self.emojis_val.get(emoji)}\n"
+
+        help_embed.add_field(
+            name="How to play",  # display messages on HOW TO PLAY
+            value=f"""Everyday, the bot will post a random word, which you will need to spookify using your creativity.
+You can rate each message according to how scary it is
+At the end of the day, the author of the message with most reactions will be the winner of the day.
+On a scale of 1 to {len(self.emojis_val)}, the reactions order:
+{emoji_message}""",
+            inline=False
+        )
+
+        # command to add spookified name
+        help_embed.add_field(
+            name="How do I add my spookified word?",
+            value=f"Just simply type `{self.bot.command_prefix}spooknamerate add my word`",
+            inline=False
+        )
+
+        await ctx.send(embed=help_embed)
+
+    @spook_name_rate.command(name='list', aliases=["all", "entries"])
+    async def list_entries(self, ctx: Context) -> None:
+        """Send all the entries up till now in a single embed."""
+        await ctx.send(embed=await self.get_responses_list(final=False))  # Send a list of users
+
+    @spook_name_rate.command(name='word')
+    async def tell_word(self, ctx: Context) -> None:
+        """Tell the current random word (name)."""
+        if not self.poll:
+            return await ctx.send(f"The name is **{self.word}**")
+        else:
+            return await ctx.send(f"The word ~~is~~ was **{self.word}**. The poll has already started, so you cannot \
+add an entry.")
+
+    @spook_name_rate.command(name="add", aliases=["+", "register"])
+    async def add_word(self, ctx: Context, *, word: str) -> None:
+        """A command that adds your word!"""
+        if not self.poll:
+            message = ctx.message  # get the message
+
+            for message_id in self.messages:
+                data = self.messages.get(message_id)
+                if data['author'] == message.author:  # if the author has already added an entry
+                    return await ctx.send(f"But you have already added an entry! Type `{self.bot.command_prefix}spooknamerate \
+delete` to delete it, and then you can add it again")
+                elif data['word'] == word:
+                    # Someone has added the word
+                    return await ctx.send("TOO LATE. Someone has already added this word.")
+
+            # otherwise
+            self.messages[message.id] = {  # store it
+                'word': word,  # store the word
+                'author': message.author,  # and the author
+                'score': 0,  # and the score
+            }
+
+            for emoji in self.emojis_val.keys():  # get the emojis
```

Comment:
```suggestion
            for emoji in self.emojis_val:  # get the emojis
```
It will implicit iterate the keys, no need to call .keys()

Reply:
I have done that, thanks for the review!